### DISCUSSION TITLE
Trip API calls scrubbed?

### DISCUSSION THREAD

**Discussion by sp9wpn** [DurgNomis-drol/mytoyota#318] [Q&A] [answered]
Hi,

I've seen that endpoint for trips data has been removed recently from the code. Is it official that this feature won't be supported?

**Comment by CM000n** [comment]
Sorry don't know what you are talking about? Trips enpoint is still in the codebase and supported.

**Comment by sp9wpn** [comment]
I've seen this commit: https://github.com/DurgNomis-drol/mytoyota/commit/38e57c0438b9216395e3ef2a667b00eedbb18e79
False alarm then?

**Reply by CM000n** [reply, marked as answer]
You seem to have misunderstood that. 😉 
This commit fixes a bug where the trips endpoint was addressed unnecessarily every time the API was called. No matter if needed or not: https://github.com/DurgNomis-drol/mytoyota/pull/298#issue-2080420088
